#固定翼控制指令的期望值		
string cmd_mode

float64 pitch_angle_sp

float64 yaw_angle_sp

float64 roll_angle_sp

float64 throttle_sp


float64 t_flight
float64 date_pitch
float64 date_pitchcmd
float64 date_roll
float64 date_rollcmd
float64 date_v
float64 date_vcmd
float64 date_h
float64 date_hcmd
float64 date_vy
float64 date_vycmd
float64 date_delta_perr
float64 date_delta_verr
float64 date_delta_gd
float64 date_yaw
float64 date_yawcmd
float64 date_flag_zw




























float64 z0

float64 z1

float64 z2

float64 z3

float64 z4